lib/gocd/monitor/analyze_support.rb

```diff
@@ -74,6 +74,18 @@ def get_hash(file_name)
       def get_log_files
         Dir[File.join(options.dest_dir, '*')].sort
       end
+
+      def are_matching?(current_stack_trace, next_stack_trace)
+        if current_stack_trace.empty? || next_stack_trace.empty?
+          return
+        end
+        if current_stack_trace.size > next_stack_trace.size
+          temp = current_stack_trace
+          current_stack_trace = next_stack_trace
+          next_stack_trace = temp
+        end
+        (current_stack_trace - next_stack_trace).empty?
```

Comment:
That doesn't exactly do it for us. Sometimes, the bottom part of the stacktraces might match, but in the later dump, the control might have moved on to do something else. Let me know if this is not clear, I could paste an example.